dandi/support/digests.py

```diff
@@ -103,22 +107,93 @@ def get_zarr_checksum(
 ) -> str:
     if path.is_file():
         return cast(str, get_digest(path, "md5"))
+    root: Tuple[str, ...]
     if basepath is None:
         basepath = path
-    dirs = {}
-    files = {}
+        root = ()
+    else:
+        root = path.relative_to(basepath).parts
```

Comment:
this code path (and block below depending on `root` being iterable) seems to not covered by a test.

Reply:
Done.